tools/static-assets/server/npm-rebuild-args.js

```diff
@@ -13,6 +13,13 @@ var args = [
   "--update-binary"
 ];
 
+// Allow additional flags to be passed via the $METEOR_NPM_REBUILD_FLAGS
+// environment variable.
+var flags = process.env.METEOR_NPM_REBUILD_FLAGS;
```

Comment:
Though I honestly don't think these two flags will ever be problematic, I've been wrong about similar things in the past. By the way, the `--build-from-source` flag that I passed for the WorkPop build is definitely not something I would be comfortable imposing on every rebuild, which is why I added the environment variable.